@uportal/open-id-connect/src/open-id-connect.js

```diff
@@ -0,0 +1,29 @@
+let bearerToken = null;
+
+export default async function openIdConnec({
```

Comment:
No "t" on the end of "Conec[t]?"

Reply:
:sweat_smile: Yep that should be `Connect`. Updated